apps/term-apply/README.md

```diff
@@ -0,0 +1,61 @@
+# term-apply
+
+term-apply is a custom SSH server used by candidates to submit their resumes
+
+## Usage
+
+term-apply is maintained and distriubuted as a [nix package](https://github.com/Nebulaworks/nix-garage/blob/master/pkgs/term-apply/default.nix) and [container](https://hub.docker.com/r/nebulaworks/term-apply).
+
+## Development
+
+These instructions recommend using `nix-shell`. If you choose not to, please make sure you have a functional `go 1.17` installation and the `make` command installed.
+
+1. You will need access to an S3 bucket and prefix(es) with the following object permissions.
+
+```
+s3:GetObject
+s3:PutObject
+s3:PutObjectTagging
+```
+
+2. From the directory where this file is located, enter a nix shell
+```
+nix-shell
+```
+
+3. Set the `TA_BUCKET` variable
+```
+export TA_BUCKET=my-bucket
+```
+
+4. If necessary, set other [environment variables](#environment-variables) for your specific environment.
+> For most usage, the defaults are fine for local development
+
+5. Create an `uploads` directory if one does not yet exist
+```
+mkdir uploads
+```
+
+6. Code
+
+7. Compile the code
+```
+make test && make build
```

Comment:
This made me realize that `GNUmake` wasnt in the nix-shell: https://github.com/Nebulaworks/orion/blob/main/apps/term-apply/shell.nix#L8-L11

I think itd be a good idea to add it for completeness: https://github.com/Nebulaworks/orion/blob/main/apps/term-apply/shell.nix#L8-L11

Reply:
nice catch, yes it should be. i mistakenly assumed it already was because of *some other project*